src/app/ApplicationTemplate.Access/ApplicationTemplate.Access.csproj

```diff
@@ -20,13 +20,13 @@
 		<PackageReference Include="Microsoft.Extensions.Http" Version="3.1.0" />
 		<PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="7.0.1" />
 		<PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="3.1.0" />
-		<PackageReference Include="Reactive.Annex" Version="0.6.0-dev.50" />
+		<PackageReference Include="Reactive.Annex" Version="2.0.0-feature.Uno5.2" />
```

Comment:
I know this is still draft, but the plan is to update to stable versions (non `-feature`) before merging this PR, right?

Reply:
I feel like It should.

Reply:
Probably just to test

Reply:
yes

Reply:
@jeanplevesque @Soap-141 does that sound good to you guys?

Reply:
Yes that works now that the target branch was changed :)
![image](https://github.com/nventive/UnoApplicationTemplate/assets/39710855/2a009c91-2d0e-4f78-914a-de5801ae852b)